bigchaindb/db/backends/rethinkdb.py

```diff
@@ -202,10 +202,9 @@ def get_asset_by_id(self, asset_id):
              .get_all(asset_id, index='asset_id')
              .concat_map(lambda block: block['block']['transactions'])
              .filter(lambda transaction:
-                     transaction['transaction']['asset']['id'] == asset_id)
+                     transaction['asset']['id'] == asset_id)
              .filter(lambda transaction:
-                     transaction['transaction']['operation'] == 'CREATE')
-             .pluck({'transaction': 'asset'}))
```

Comment:
everything will work the same with or without pluck because the structure of the document is maintained. The only difference is that all other keys are empty